components/servo/lib.rs

```diff
@@ -130,6 +130,34 @@ mod media_platform {
     use super::ServoMedia;
     use servo_media_gstreamer::GStreamerBackend;
 
+    #[cfg(target_os = "windows")]
+    fn set_gstreamer_log_handler() {
+        use gstreamer::{debug_add_log_function, debug_remove_default_log_function, DebugLevel};
+
+        debug_remove_default_log_function();
+        debug_add_log_function(|cat, level, file, function, line, _, message| {
```

Comment:
You probably also want to include the name of the object in the logs somewhere. The `_` parameter :)